app/models/foreman_kubevirt/kubevirt.rb

```diff
@@ -142,6 +148,7 @@ def create_vm(args = {})
       # Add image as volume to the virtual machine
       image_provision = args["provision_method"] == "image"
       if image_provision
+        verify_booting_from_image_is_possible(args)
```

Comment:
so if we don't anything else from args, except of volumes_attributes, we can pass to this method `volumes_attributes` that was set to a variable on line 145.